plugins/catalog-backend/src/catalog/DatabaseEntitiesCatalog.ts

```diff
@@ -14,12 +14,15 @@
  * limitations under the License.
  */
 
-import { Entity } from '@backstage/catalog-model';
-import { Database } from '../database';
-import { EntitiesCatalog, EntityFilters } from './types';
+import type { Entity, EntityPolicy } from '@backstage/catalog-model';
+import type { Database, DbEntityResponse, EntityFilters } from '../database';
+import type { EntitiesCatalog } from './types';
```

Comment:
I don't think we actually need to use type-only imports, but it doesn't hurt :>

Reply:
yeah it kinda became a habit of sorts... at least when i remember :)